learntools/core/utils.py

```diff
@@ -31,6 +40,21 @@ def bind_exercises(g, exercises, start=1, var_format='q{n}'):
     corresponding variable in the sequence is skipped over. e.g. [SpamProblem, None,
     EggsProblem], will generate variables q1 and q3.
     """
+    all_problem_classes = []
+    for thing in exercises:
+        if isinstance(thing, list):
```

Comment:
Whoops, this actually isn't necessary. I'll take it out. I was thinking I had implemented the idea in the TODO below (passing sublists of problem classes instead of explicitly instantiating and passing `MultipartProblem` instances) - but no, it's still just a todo.